editor/editor_log.h

```diff
@@ -115,6 +116,7 @@ class EditorLog : public HBoxContainer {
 	};
 
 	Vector<LogMessage> messages;
+	RegEx resource_path_regex = RegEx();
```

Comment:
```suggestion
	RegEx resource_path_regex;
```

I don't think you need to explictly call the default constructor :)